force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -40,17 +40,28 @@ public with sharing class ClassesObjectsMethods {
      */
     public static List<String> practiceStringMethods() {
         // Initialize a String with 'Hello, Salesforce!'
+        String hello = 'Hello, Salesforce!';
         
         // Get the length of the String        
-        // Get a substring of the String of the first 5 characters
+        Integer lenHello = hello.length();
+        // Get a substring of the String o the first 5 characters
+        String subStringHello = hello.substring(0,5);
         // Convert the String to lower case
+        String lowerHello = hello.toLowerCase();
+
         
         // Return the list with the string data
+        List<String> helloList = new List<String>();
+
         // In the first position add the string length
+        String lenHelloString = String.valueOf(lenHello);
+        helloList.add(lenHelloString);
         // In the second position add the substring
+        helloList.add(subStringHello);
         // In the third position add the string in lower case
+        helloList.add(lowerHello);
 
-        return null; // Replace null with the variable you used to store the result
+        return helloList; // Replace null with the variable you used to store the result
```

Comment:
Something I encourage students to think about is, "Do I need a variable here?". In this case, you could add the results of your expressions directly to the list without losing any readability. 

```
 public static List<String> practiceStringMethods() {
        List<String> strings = new List<String>();
        String helloSalesforce = 'Hello, Salesforce!';
        strings.add(String.valueOf(helloSalesforce.length()));
        strings.add(helloSalesforce.substring(0, 5));
        strings.add(helloSalesforce.toLowerCase());
        return strings;
    }
 ```